metabase/metabase
# allow saving chart images

Closes https://github.com/metabase/metabase/issues/4701

### Description

A last-minute change to allow users to save png images of charts

### How to verify

1. Create a line/area/bar/pie/gauge/... chart and ensure you can save it
2. Add the charts on a dashboard and ensure you can save images from dashcards

Saving images does not work on visualizations that have `canSavePng=false` such as tables, etc.

### Demo

#### QB:
<img width="1146" alt="Screen Shot 2023-02-22 at 3 17 42 PM" src="https://user-images.githubusercontent.com/14301985/220720140-d3f66a3e-748b-4544-95a5-bf562677f8bf.png">

#### Dashcards:
<img width="910" alt="Screen Shot 2023-02-22 at 3 18 22 PM" src="https://user-images.githubusercontent.com/14301985/220720277-d4e98bb4-6eee-4ef5-8853-dbc7d07103b7.png">


### Checklist
